### DISCUSSION TITLE
Layout/Class error associated with TOC and Title on new server

### DISCUSSION THREAD

**Discussion by DuncanLHS** [requarks/wiki#6071] [Error / Bug Report]
New build server v2.5.295
Docker(Linux)

Newly installed server. Found a layout issue with Title being overlapped by the TOC. Margin has not been applied to the Title element. No custom CSS injected in Theme or Page.

Title here is "Home" but is hidden by the TOC:
![image](https://user-images.githubusercontent.com/16748684/213592615-836b9922-c866-434f-bb65-768ece0a0a66.png)

Problem appears to be with classes applied to the element:

Title incorrect element found to be this:
`<div class='page-col-content is-page-header offset-lg-0 offset-xl-0 col pl-4'
    ...`

Whereas on another Wiki.js site the Title element was found to be:
`<div class="page-col-content is-page-header offset-lg-3 offset-xl-2 col pl-4"
    ...`

I tracked it to a @media query in a style element hard coded in each Page html. The class `offset-lg-3` applies the required `margin-left: 25%` styling whereas `offset-lg-0` applies `margin: 0`

This can be corrected by switching the TOC position to "Right" and then back to "Left" in the Theme admin panel, the correct class is restored to the element. So assume this is an issue only with new servers, i.e. the 'left' tocposition not being applied to the title element from new. I don't know Vue well enough to track it down but suggest f3133a72ec40e49a5115e9cfd543cdd1a9e2e378

Possibly `tocPosition` is null at https://github.com/requarks/wiki/blob/main/client/themes/default/components/page.vue#L54 when a new instance of the server is created??

**Comment by white-gecko** [comment]
I can confirm this bug with a fresh installation of the docker container from https://fleet.linuxserver.io/image?name=linuxserver/wikijs tag 2.5.295 on podman.
